docker-entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$*" == npm*start* ]]; then
 	if [ ! -e "$GHOST_CONTENT/config.js" ]; then
 		sed -r '
 			s/127\.0\.0\.1/0.0.0.0/g;
-			s!path.join\(__dirname, (.)/content!path.join(process.env.GHOST_CONTENT, \1!g;
+			s!path.join(__dirname, \(.\)/content!path.join(process.env.GHOST_CONTENT, \1!g;
```

Comment:
You are switching the capturing block to literal parentheses and the literal parenthesis to the start of a capture block.

It currently works correctly here: 
```console
$ docker run -d --name gh ghost
$ docker exec -it gh bash
root@80a835f27b97:/usr/src/ghost# grep -n '/content/' config.example.js
19:                filename: path.join(__dirname, '/content/data/ghost.db')
61:                filename: path.join(__dirname, '/content/data/ghost-dev.db')
76:            contentPath: path.join(__dirname, '/content/')
90:                filename: path.join(__dirname, '/content/data/ghost-test.db')
root@80a835f27b97:/usr/src/ghost# grep -n 'path.join(process' $GHOST_CONTENT/config.js
19:                filename: path.join(process.env.GHOST_CONTENT, '/data/ghost.db')
61:                filename: path.join(process.env.GHOST_CONTENT, '/data/ghost-dev.db')
76:            contentPath: path.join(process.env.GHOST_CONTENT, '/')
90:                filename: path.join(process.env.GHOST_CONTENT, '/data/ghost-test.db')
```

How did you start your container? Did you provide a custom `config.js`?